src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -81,12 +82,22 @@ const ArticlePageMostReadSection = styled(MostReadSection)`
 
 const ArticlePage = ({ pageData, mostReadEndpointOverride }) => {
   const { articleAuthor } = useContext(ServiceContext);
+  const { state } = useLocation();
+  const hash = state?.hash;
   const headline = getHeadline(pageData);
   const description = getSummary(pageData) || getHeadline(pageData);
   const firstPublished = getFirstPublished(pageData);
   const lastPublished = getLastPublished(pageData);
   const aboutTags = getAboutTags(pageData);
 
+  useEffect(() => {
+    if (hash) {
+      const element = document.getElementById(hash);
+      element?.scrollIntoView();
```

Comment:
Please could you explain what 'scrollIntoView' and the line below does?

Reply:
When a link ends in a hash for example `https://www.test.bbc.com/mundo/articles/ce4krqk1334o#Anchor-1`, `element?.scrollIntoView()` will scroll do to the header with id `Anchor-1` and `element?.focus()` will ensure screen readers skip on this header.